packages/atlas/test/atlas_test.dart

```diff
@@ -47,8 +47,10 @@ main() {
         initialCameraPosition: initialCameraPosition,
         markers: Set<Marker>(),
         circles: Set<Circle>(),
+        polylines: Set<Polyline>(),
         showMyLocation: false,
         showMyLocationButton: false,
+        showTraffic: false,
```

Comment:
Instead of `showTraffic` being a boolean, I think it'd make more sense to implement a `mapOverlay` enum. This way, when we can continue to add overlay functionality for bike paths, transit, etc... we aren't continuously adding booleans. 
cc: @tchabot22